Studentify/Models/Authentication/LoginModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Studentify.Models.Authentication
+{
+    public class LoginModel
+    {
+        [Required(ErrorMessage = "User Name is required")]
```

Comment:
To się będzie wyświetlać z tego co rozumiem wtedy, kiedy w body nie będzie tego atrybutu. Także może by się przydał ErrorMessage bardziej w stylu co zrobić, żeby to naprawić (coś że body musi mieć ten parametr, a go nie ma). Ale no to na razie w sumie nie do końca chyba nie jesteśmy pewni co to jest, to może chyba zostać. ( tak samo pozostałe atrybute w register modelu też)

Reply:
Tego chyba nie da się w tak łatwy sposób określić. Później pewnie będziemy jeszcze zmieniać te atrybuty to na sam koniec można będzie dodać jakieś informacje od validatora